pkg/manager/impl/execution_manager_test.go

```diff
@@ -3922,7 +4063,29 @@ func TestGetExecutionConfigOverrides(t *testing.T) {
 		assert.Equal(t, rmLabels, execConfig.GetLabels().Values)
 		assert.Equal(t, rmAnnotations, execConfig.GetAnnotations().Values)
 	})
-	t.Run("matchable resource partial config", func(t *testing.T) {
+	t.Run("launchplan with no config except deprecated auth role but having matchable attribute", func(t *testing.T) {
```

Comment:
also not sure about this case, if the launch plan has any overridable field set in the spec then shouldn't we get all the execution config attributes from the LP and not the matchable attribute?

Reply:
Again the same arg here too i was thinking , 
If the user already is aware of using security context in one place then can we expect him to use in others too ?